pgtype/uint64_test.go

```diff
@@ -0,0 +1,21 @@
+package pgtype_test
+
+import (
+	"context"
+	"testing"
+
+	"github.com/jackc/pgx/v5/pgtype"
+	"github.com/jackc/pgx/v5/pgxtest"
+)
+
+func TestUint64Codec(t *testing.T) {
+	pgxtest.RunValueRoundTripTests(context.Background(), t, defaultConnTestRunner, pgxtest.KnownOIDQueryExecModes, "pg_lsn", []pgxtest.ValueRoundTripTest{
```

Comment:
This needs to skip CockroachDB as well.

But it also makes me wonder if this is actually testing `Uint64Codec`. It's using the `pg_lsn` type. That would mean that `PgLSNCodec` is being used. We'd need to use another real uint64 PG type for this test... Hmm... That makes me wonder again about whether Uint64 should actually exist. I looked in the  PostgreSQL `pg_type` table and your example of `FullTransactionId` isn't exposed there. I don't think there are any other uint64 types in PostgreSQL. (Run `select * from pg_type where typlen = 8;` in PostgreSQL to see all candidates).

Unless we can find another uint64 type in PostgreSQL I think pg_lsn is all that should exist.

Reply:
> I don't think there are any other uint64 types in PostgreSQL. (Run select * from pg_type where typlen = 8; in PostgreSQL to see all candidates).

Thanks. I looked at source code of `macaddr8` and `xid8` and confirmed that they are not backed with `uint64`.

> Unless we can find another uint64 type in PostgreSQL I think pg_lsn is all that should exist.

I am convinced now and am closing this pull request and the issue [Add pgtype.Uint64 to support pg_lsn type · Issue #1407 · jackc/pgx](https://github.com/jackc/pgx/issues/1407).

Sorry for taking your time and thanks again for your review.